manifests/path.pp

```diff
@@ -12,7 +12,7 @@
   $directories           = 'MANDATORY',
 ) {
 
-  include env
+  include ::env
```

Comment:
Hi Phil,

Why the double colon in the include? I didn't find any reason for it in the documentation.
https://puppet.com/docs/puppet/5.3/lang_classes.html#using-include
https://puppet.com/docs/puppet/5.5/lang_namespaces.html

Thanks

Reply:
for more details please check https://docs.puppet.com/puppet/3/lang_namespaces.html#relative-name-lookup-and-incorrect-name-resolution

Reply:
ok good

Reply:
the PR's message is misleading!